aten/src/ATen/native/RNN.cpp

```diff
@@ -360,15 +365,65 @@ struct QuantizedCellParamsDynamic : public CellParamsBase {
         /*w_ih_packed=*/std::move(packed_params[0]),
         /*w_hh_packed=*/std::move(packed_params[1]),
         /*bias_ih=*/std::move(tensors[0]),
-        /*bias_hh=*/std::move(tensors[1]));
+        /*bias_hh=*/std::move(tensors[1]),
+        /*reduce_range=*/false);
+  }
+};
+
+struct QuantizedCellParamsDynamicReduceRange : public QuantizedCellParamsDynamic {
```

Comment:
Do we still need quantizedCellParamsDynamic? Why not replace that with this struct?

Reply:
Older models are serialized with QuantizedCellParamsDynamic so we need to keep it around for BC.

Reply:
Dont follow, this is not an aten op, so why does this affect serialization?